zzk/service/hoststate.go

```diff
@@ -185,6 +185,16 @@ func (l *HostStateListener) Spawn(shutdown <-chan interface{}, stateID string) {
 			logger.WithError(err).Error("Could not watch host state")
 			return
 		}
+		// set up a listener on the service state node
+		ssdat = &ServiceState{}
+		ssevt, err := l.conn.GetW(sspth, ssdat, done)
```

Comment:
use ExistsW, instead of GetW because, we only want an event if the node is deleted, not if it is updated.